File: 250424/코드네임/code-name.cpp
#include <iostream>
#include <string>
using namespace std;

class User {
    public:
        char name;
        int score;
        User(char name, int score) {
            this->name = name;
            this->score = score;
        }
        User() {}
};

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    User user[5];
    char name;
    int score;
    for(int i = 0; i < 5; i++) {
        cin >> name >> score;
        user[i] = User(name, score);
    }
    int index = 0;
    for(int i = 1; i < 5; i++) {
        if(user[index].score > user[i].score) {
            index = i;
        }
    }
    cout << user[index].name << " " << user[index].score;
    return 0;
}

File: 250424/코드네임/code-name.java
import java.util.Scanner;

class Score {
    String name;
    int score;
    public Score() {
        this.name = " ";
        this.score = 0;
    }
    public Score(String name, int score) {
        this.name = name;
        this.score = score;
    }
}
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int index = -1;
        Score[] scores = new Score[5];
        for(int i = 0; i < 5; i++) {
            String name = sc.next();
            int score = sc.nextInt();
            scores[i] = new Score(name, score);
        }
        int minScore = Integer.MAX_VALUE;
        for(int i = 0; i < 5; i++) {
            if(scores[i].score < minScore) {
                minScore = scores[i].score;
                index = i;
            }
        }
        System.out.println(scores[index].name + " " + scores[index].score);
        sc.close();
    }
}